doc/rest.md

```diff
@@ -0,0 +1,103 @@
+Introduction
+==============
+Jitsi-videobridge supports two HTTP(S) interfaces, a _public_ and a _private_ one. 
+The two interfaces use different ports. The _private_ interface exposes
+HTTP endpoints which are not meant to be publicly accessible, such as:
```

Comment:
maybe it's worth referencing that they are meant to be accessed by other components of your deployment (like signaling server, etc.)?